templates/static/js/src/demobox/components/LiveDemoBox/LiveDemoBox.tsx

```diff
@@ -34,28 +40,42 @@ const LiveDemoBox = () => {
     fetchData();
   }, []);
 
+  const handleChange = (e: React.ChangeEvent<HTMLSelectElement | HTMLInputElement>) => {
+    if(e.target.value === "on"){
+      setSelectValues({...selectValue, [e.target.name] : !selectValue[e.target.name]} )
+    } else {
+      setSelectValues({...selectValue, [e.target.name] : e.target.value.toLowerCase()} )
+    }
+  }
+
+  const url =`/docs/examples/patterns/notifications/toast?type=${selectValue.type}&style=${selectValue.style}&actions=${selectValue.actions}&dismiss=${selectValue.dismiss}&timestamp=${selectValue.timestamp}`
+
   return (
     <section className="p-strip--light">
       {configValues && configValues.dropdown && configValues.switch && (
         <form>
           <div className="row" style={{gridGap: 0}}>
             <div className="col-2">
-              <Select id="select-type" label="select-type" name="Type" options={configValues.dropdown.type} />
+              <Select id="type" label="Type" name="type" onChange={handleChange} options={configValues.dropdown.type} />
             </div>
             <div className="col-2">
-              <Select id="select-theme" label="select-theme" name="Theme" options={configValues.dropdown.theme} disabled />
+              <Select id="theme" label="Theme" name="theme" onChange={handleChange} options={configValues.dropdown.theme} disabled />
             </div>
             <div className="col-2">
-              <Select id="select-style" label="select-style" name="Style" options={configValues.dropdown.style} />
+              <Select id="style" label="Style" name="style" onChange={handleChange} options={configValues.dropdown.style}/>
             </div>
           </div>
           <div className="row" style={{gridGap: 0}}>
             <div className="p-card col-6">
               <h3>Example</h3>
-              <p className="p-card__content">space for exmaple here</p>
+              <p className="p-card__content">
+              <div>
+                <iframe src={url} width="100%" height="200"></iframe>
```

Comment:
I believe the original `example.js` calculates the height based on the size of the element? If it's fixed at 200, elements that have more than this height will require scrolling

Reply:
Thanks @carkod for your review! Yep I will update the height 👍